docs/snippets/modules/model_io/models/llms/how_to/llm_caching.mdx

```diff
@@ -14,7 +14,7 @@ from langchain.cache import InMemoryCache
 langchain.llm_cache = InMemoryCache()
 
 # The first time, it is not yet in cache, so it should take longer
-llm("Tell me a joke")
+llm.predict("Tell me a joke")
```

Comment:
Why is this changing?

Reply:
to use the consist predict/predict_messages interface